app/src/pages/Main/HelpOfferMarker/index.js

```diff
@@ -19,6 +19,24 @@ export default function HelpsMarker({ helpOffer }) {
         helpOffer.helpedUserId?.includes(user._id) ||
         helpOffer.possibleHelpedUsers?.some((obj) => obj._id === user._id);
 
+    const renderMarkerText = () => {
+        return userIsOwner || userIsParticipating ? (
+            <Text style={styles.calloutTitle} numberOfLines={1}>
+                {userIsOwner ? 'Sua oferta' : 'Participando'}
+            </Text>
+        ) : (
+            <>
+                <Text style={styles.calloutTitle} numberOfLines={1}>
+                    Oferta de ajuda
+                </Text>
+                <Text style={styles.calloutPersonName} numberOfLines={1}>
+                    {helpOwnerNameFormated}
+                </Text>
+                <Text style={styles.calloutPress}>Toque para ver</Text>
+            </>
+        );
+    };
```

Comment:
Sugestão apenas para deixar o código mais legível, não obrigatória.
```suggestion
const getTitle = () => {
    if (userIsParticipating) return 'Participando';
    else if (userIsOwner) return 'Sua oferta';
    return 'Oferta de ajuda';
};

const renderMarkerText = () => {
    const title = getTitle();
    const showBody = !userIsOwner && !userIsParticipating;
    return (
        <>
            <Text style={styles.calloutTitle} numberOfLines={1}>
                {title}
            </Text>
            {showBody && (
                <>
                    <Text style={styles.calloutPersonName} numberOfLines={1}>
                        {helpOwnerNameFormated}
                    </Text>
                    <Text style={styles.calloutPress}>Toque para ver</Text>
                </>
            )}
        </>
    );
};
```